src/ape/contracts/base.py

```diff
@@ -699,17 +699,12 @@ def poll_logs(
             start_block = height + 1
```

Comment:
Oh, is line 691 a problem? It seems like would prevent `0` confirmations from working.